e2e/helpers/testUser.ts

```diff
@@ -82,6 +75,11 @@ export async function loginTestUser(
   await page.getByLabel('Password').fill(testUserPassword)
   await page.getByRole('button', { name: 'Sign in' }).click()
 
+  // Wait for redirect
+  await expect(async () => {
+    expect(page.url()).not.toContain('LoginPlace')
+  }).toPass()
+
```

Comment:
As Lawrence suggested, wait for the login redirect so if a test fails, we can more easily tell on which page the failure occurred. 

Per [the docs](https://playwright.dev/docs/api/class-frame#frame-wait-for-navigation), `waitForNavigation` is inherently racy and is deprecated in favor of `waitForURL`. However, since SWC login redirects to different pages depending on the last place, we may not know the URL of the redirect. Instead, we can wait for the URL to change from the LoginPlace to another value.